course-instructions.md

```diff
@@ -0,0 +1,342 @@
+
+
+### :tv: Activity: watch video on targets from at least 7:20 to 11:05 (you are welcome to watch the full talk if you'd like)
+
+Let your course labmate know what contrasts you identified between solutions in `make` and what is offered in R-specific tools, like `targets`. 
+
+</details>
+
+<hr>
+
+<details>
+<summary><h2>The Anatomy of a Makefile</h2></summary>
+
+Our `targets` pipelines in R use a `makefile` file to orchestrate the connections among files, functions, and phases. In this issue, we're going to develop a basic understanding of how these files work, starting with the anatomy of the `_targets.R` file.
+
+### Setting up a `targets` pipeline
+
+In addition to phases (which we covered earlier), it is important to decompose high-level concepts (or existing scripts) into thoughtful functions and "targets" that form the building blocks of data processing pipelines. A target is a noun we use to describe a tangible output of a function, which is often a file or an R object. Targets can be used as an end-product (like a summary map) or as input _into_ another function to create _another_ target. 
+
+To set up a `targets` pipeline, you will need to create the base makefile named `_targets.R` that will declare and orchestrate the rest of the pipeline connections.
+
+---
+A simple version of `_targets.R` might look something like this:
+
+```r
+library(targets)
+source("code.R")
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+
+list(
+  tar_target(
+    model_RMSEs_csv,
+    download_data(out_filepath = "model_RMSEs.csv"),
+    format = "file"
+  ), 
+  tar_target(
+    eval_data,
+    process_data(in_filepath = model_RMSEs_csv),
+  ),
+  tar_target(
+    figure_1_png,
+    make_plot(out_filepath = "figure_1.png", data = eval_data), 
+    format = "file"
+  )
+)
+```
+
+This file defines the relationships between different "targets" (see how the target `model_RMSEs_csv` is an input to the command that creates the target `eval_data`?), tells us where to find any functions that are used to build targets (see the `source` call that points you to `code.R`), and declares the package dependencies needed to build the different targets (see the `target_option_set()` command that passes in a vector of packages). 
+
+We'll briefly explain some of the functions and conventions used here. For more extensive explanations, visit the [`targets` documentation](https://books.ropensci.org/targets/walkthrough.html). 
+
+  - As you would with normal R scripts, put any `source` commands for loading R files and `library` commands for loading packages at the top of the file. The packages loaded here should be only those needed to build the targets _plan_; packages needed to build specific _targets_ can be loaded later.
```

Comment:
"As you would with normal R scripts" is making assumptions about the trainee and what is "normal"

Reply:
I do think this falls into the category of "normal" - esp. with a "lazy loading" language like R, however I see your point and have removed that assumption.